packages/AlertDialog/index.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {useState} from 'react';
+import Modal from 'react-native-modalbox';
+import styled from '@emotion/native';
+
+import {StyleProp, ViewStyle} from 'react-native';
+
+const ContentContainer = styled.SafeAreaView`
+  flex-grow: 1;
+  flex-shrink: 1;
```

Comment:
Is there any reason not using `flex: 1` instead?

Reply:
Recently, there was an issue where UI appears differently by browser due to the change of policy that interprets `flex : 1` as [chrome update](https://chromium.googlesource.com/chromium/src/+log/91.0.4472.164..92.0.4515.107?pretty=fuller&n=10000). I didn't use abbreviation to prevent similar issues from occurring in the future.